ui/package.json

```diff
@@ -21,13 +21,14 @@
     "react-dom": "^18.2.0",
     "react-router-dom": "^6.3.0",
     "react-table": "^7.7.0",
-    "swr": "^1.3.0"
+    "swr": "^1.3.0",
+    "universal-cookie": "^4.0.4"
```

Comment:
Possible to run `npm install` in the `ui` directory – there should be an update to `package-lock.json` as well